pkg/ui/markdown/styles.go

```diff
@@ -0,0 +1,208 @@
+package markdown
+
+// DefaultStyle defines the default Atmos markdown style
+var DefaultStyle = []byte(`{
```

Comment:
Since this is just a string, what about moving this to the `style` section in `atmos.yaml`, and setting some defaults built-in.

Reply:
resolved on latest version